README.md

```diff
@@ -11,50 +11,57 @@ Adds a server side check for a mandatory app update and server-side configuratio
 
 The preferred method is via CocoaPods:
 
-    pod 'ManUp'
-
+```pod
+pod 'ManUp'
+```
 
 ## Usage
 
 ManUp will download a ManUp configuration file (json) that is hosted on a server of your choice. This file will have the current app store version, the minimum version, and a URL to the app store or app website.
 
-    {
-        "ios": {
-            "url": "https://itunes.apple.com/app/id0000000?mt=8",
-            "latest": "2.0",
-            "minimum": "1.1",
-            "enabled": true
-        }
-    }
+```json
+{
+  "ios": {
+    "url": "https://itunes.apple.com/app/id0000000?mt=8",
+    "latest": "2.0",
+    "minimum": "1.1",
+    "enabled": true
+  }
+}
+```
 
 Running ManUp will download this file and compare it to the installed app's version to determine if there is an update available (`latest`), or if there is a mandatory update required (`minimum`).
 
 #### Swift
 
+```swift
 	@import ManUp
```

Comment:
n.b. this issue is fixed in my other PR https://github.com/NextFaze/ManUp/pull/22